src/utils/utils.ts

```diff
@@ -321,3 +322,11 @@ export const batchDetailsOptionalDisplay = (data: string) => {
       return detailsPageOptionalDisplay(data);
   }
 };
+
+export const toastifyCustomStyle: ToastOptions<{}> = {
+  hideProgressBar: true,
```

Comment:
Are we setting this for every toast?  if so we should just add it to the ToastContainer props instead of passing it in as toast options instead.

Reply:
We are not using ToastContainer as it was giving multiple toast. so just passing with toast.success and toast.error